creditnote.go

```diff
@@ -61,10 +61,12 @@ type CreditNoteLineParams struct {
 // its amount_due. For a status=paid invoice, a credit note does not affect its amount_due. Instead, it can result
 // in any combination of the following:
 //
+//
 // Refund: create a new refund (using refund_amount) or link an existing refund (using refund).
 // Customer balance credit: credit the customer's balance (using credit_amount) which will be automatically applied to their next invoice when it's finalized.
 // Outside of Stripe credit: record the amount that is or will be credited outside of Stripe (using out_of_band_amount).
 //
+//
```

Comment:
Nit: Not sure why these spaces were added

Reply:
Looks like this is just reverting changes from #1547